src/backend/aspen/api/views/auth.py

```diff
@@ -20,21 +20,23 @@
 @router.get("/login")
 async def login(
     request: Request,
-    auth0: StarletteRemoteApp = Depends(get_auth0_client),
+    auth0: StarletteOAuth2App = Depends(get_auth0_client),
     settings: Settings = Depends(get_settings),
 ) -> Response:
     return await auth0.authorize_redirect(request, settings.AUTH0_CALLBACK_URL)
 
 
 @router.get("/callback")
 async def auth(
-    request: Request, auth0: StarletteRemoteApp = Depends(get_auth0_client)
+    request: Request,
+    auth0: StarletteOAuth2App = Depends(get_auth0_client),
+    settings: Settings = Depends(get_settings),
 ) -> Response:
     try:
         token = await auth0.authorize_access_token(request)
     except OAuthError:
         raise ex.UnauthorizedException("Invalid token")
-    userinfo = token.get("userinfo")
+    userinfo = token["userinfo"]
```

Comment:
Why was this changed? We'll return a 500 error if our access token is missing user info here.

Reply:
Isn't that what we want? Wouldn't a user just get silently redirected to the splash page otherwise?

Reply:
ehhh, I'd like to at least control the exception if possible. Return an UnauthorizedException then?

Reply:
Done!